staff/meetings/minutes

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import os
+import meetings
+from ocflib.misc.shell import get_editor
+from sys import exit
+from datetime import datetime
+from subprocess import call, check_output
```

Comment:
check_output import unused

pre-commit should tell you about these if you run it (make test)